client/src/js/samples/components/__tests__/SampleFileSizeWarning.test.js

```diff
@@ -0,0 +1,39 @@
+import { SampleFileSizeWarning, mapStateToProps } from "../SampleFileSizeWarning";
+
+describe("<SampleFileSizeWarning />", () => {
+    let props;
+    beforeEach(() => {
+        props = {
+            show: true
+        };
+    });
+
+    it("should render when [show=true]", () => {
+        const wrapper = shallow(<SampleFileSizeWarning {...props} />);
+        expect(wrapper).toMatchSnapshot();
+    });
+    it("should render when [show=false]", () => {
+        props.show = false;
+        const wrapper = shallow(<SampleFileSizeWarning {...props} />);
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("mapStateToProps", () => {
+    let state = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['state' is never reassigned. Use 'const' instead. (prefer-const)](https://app.codacy.com/app/virtool/virtool/pullRequest?prid=4726348)